index.js

```diff
@@ -49,21 +54,25 @@ app.get("/server-config.js", (req, res) => {
 });
 
 // bulk data implementation
-app.use(["/:sim/fhir", "/fhir"], bulkData);
+router.use(["/:sim/fhir", "/fhir"], bulkData);
 
 // static files
-app.use(express.static("static"));
+router.use(express.static("static"));
 
 // global error handler
-app.use(function (err, req, res, next) {
+router.use(function (err, req, res, next) {
     console.error(err.stack);
     res.status(500).send('Something broke!');
 });
 
 if (!module.parent) {
+    app.use('/bulk-data-server', router);
+
     app.listen(config.port, function() {
-        console.log("Server listening at " + config.baseUrl);
+        console.log("Server listening  at /bulk-data-server" );
```

Comment:
why not print out `config.baseUrl` anymore?  seems useful to have the full thing.

Reply:
I'll print out the config.baseUrl, but it may not be correct depending on if the user provided the BASE_URL env var

Reply:
made update